conda.recipe/meta.yaml

```diff
@@ -1,16 +1,19 @@
 package:
   name: bcolz 
-  version: "0.7.2"
+
+source:
+    git_rev: v1.7.0
+    git_url: https://github.com/pypa/setuptools_scm.git
 
 build:
-    number: {{environ.get('BINSTAR_BUILD', 1)}}
+    preserve_egg_dir: True
 
 requirements:
   build:
     - python
+    - setuptools_scm 
```

Comment:
sharp, thanks!
No luck though, I tried in  6ccc459 but still failing, I think this package might be missing in the default channel